CodeLite/clINIParser.cpp

```diff
@@ -27,7 +27,7 @@ void clINISection::AddKeyValue(const wxString& key, const wxString& value)
     m_entries.push_back(kv);
 }
 
-void clINISection::Serialise(std::ostream& string) const
```

Comment:
This one is actually the English way of writing it "Serialise", "Initialise" 😉 , some will argue that your fix is the wrong one (me included)

Reply:
:)
All others places use `Serialize`.
Didn't check Initiali[sz]e.